README.md

```diff
@@ -44,30 +44,37 @@ The currently active members of the [Social Team](./social-team.md):
 
 - [@bnb](https://github.com/bnb) - **Tierney Cyren**
 
-## Node.js Foundation Calendar
-A calendar of The Foundation's meetings can be found at:
+## Node.js Project Calendar
+
+A calendar of the project's meetings can be found at:
 
 https://nodejs.org/calendar
 
 <sup>Click `+GoogleCalendar` at the bottom right to add to your own Google calendar.</sup>
 
-The calendar is maintained by:
+All calendar maintainers have `Make changes AND manage sharing` permissions. If you would like to help maintain your team's calendar events, open a PR adding your name to the list above. Once approved, one of the calendar maintainers will add you to the calendar settings.
+
+This list should be reviewed and pruned annually (at minimum). The calendar has a yearly recurring event on Jan 31st for this. An issue should be opened asking the calendar maintainers for their continued volunteering efforts (directly @-mention all members). After 1 week, this list should be PRed removing members that did not respond. [The calendar permissions to be updated once merged](https://support.google.com/a/answer/117596?hl=en).
+
+### Calendar Maintainers
+
 <!-- sorted by GitHub handle -->
-- [@amiller-gh](https://github.com/amiller-gh) - **Adam Miller**
-- [@bamieh](https://github.com/bamieh) - **Ahmad Bamieh**
 - [@bethgriggs](https://github.com/bethgriggs) - **Beth Griggs**
 - [@bnb](https://github.com/bnb) - **Tierney Cyren**
-- [@dshaw](https://github.com/dshaw) - **Dan Shaw**
 - [@gibfahn](https://github.com/gibfahn) - **Gibson Fahnestock**
-- [@hackygolucky](https://github.com/hackygolucky) - **Tracy Hinds**
-- [@maddhruv](https://github.com/maddhruv) - **Dhruv Jain**
 - [@mcollina](https://github.com/mcollina) - **Matteo Collina**
 - [@mhdawson](https://github.com/mhdawson) - **Michael Dawson**
 - [@MylesBorins](https://github.com/MylesBorins) - **Myles Borins**
 - [@ryanmurakami](https://github.com/ryanmurakami) - **Ryan Lewis**
 - [@trott](https://github.com/trott) - **Rich Trott**
-- [@williamkapke](https://github.com/williamkapke) - **William Kapke**
 
-All calendar maintainers have `Make changes AND manage sharing` permissions. If you would like to help maintain your team's calendar events, open a PR adding your name to the list above. Once approved, one of the calendar maintainers will add you to the calendar settings.
+### Emeritus Calendar Mainters
```

Comment:
```suggestion
### Emeriti Calendar Maintainers
```

Is an emeriti list really called for here? Seems like this could be omitted.

Reply:
Happy to omit if folks want, it’s just something I’ve found useful in lowering barriers to getting things merged uncontroversially when it comes to membership-y PRs.

Reply:
If you do keep it, please correct the spelling of `Maintainers`. I'm still in favor of removing it. But it's fine by me to leave it in if I'm the only one who feels that way!